app/src/main/java/org/akvo/flow/api/S3Api.java

```diff
@@ -1,3 +1,22 @@
+/*
+ *  Copyright (C) 2013 Stichting Akvo (Akvo Foundation)
```

Comment:
Why does it say 2013 here? If you just added this copyright notice, should it be 2017? or 2010-2017